cypress/e2e/folders.spec.ts

```diff
@@ -353,7 +353,7 @@ describe("Folders flow", () => {
     })
   })
 
-  describe("Create page, delete page, edit page settings in subfolder", () => {
+  describe.only("Create page, delete page, edit page settings in subfolder", () => {
```

Comment:
Can we remove the only here?

Reply:
good catch! missed this out when pushing to origin ._.